lib/components/form/default-search-form.js

```diff
@@ -1,19 +1,25 @@
-import React, { Component } from 'react'
 import PropTypes from 'prop-types'
+import React, { Component } from 'react'
 
 import LocationField from './connected-location-field'
-import SwitchButton from './switch-button'
 import TabbedFormPanel from './tabbed-form-panel'
+import SwitchButton from './switch-button'
 import defaultIcons from '../icons'
 
+// Use the icons props to override icons for certain modes in the mode selector panel.
+// If no icon is provided for a specific mode, the default OTP-ui icon will be used.
+// const customIcons = {
+//  TRANSIT: <TriMetModeIcon label='rail' />
+// }
+
```

Comment:
Isn't this possible with `config.js` in trimet-mod-otp as well? Maybe add a TODO statement here about consolidating that repo/code into otp-rrr

Reply:
Correct, this is a TODO item that is related to #150.

Reply:
I'm removing this comment for this PR.